src/lib/crypto/s2k.cpp

```diff
@@ -179,10 +179,11 @@ pgp_s2k_compute_iters(pgp_hash_alg_t alg, size_t desired_msec, size_t trial_msec
 
     pgp_hash_finish(&hash, buf);
 
-    const uint64_t duration = end - start;
-
-    if (duration == 0)
+    uint64_t      duration = end - start;
+    const uint8_t MIN_ITERS = 96;
```

Comment:
Just a note that I forgot to mention but typically the "don't declare variables until needed" guideline doesn't apply to constants, so this one could just as easily be at the top.

C++ Coding Standards: 101 Rules, Guidelines, and Best Practices
```
Declare variables as locally as possible.
...
Because constants don't add state, this Item does not apply to them.
```

(I personally don't mind either way, as we have no official guideline on it)